lib/ash/domain/verifiers/validate_arguments_to_code_interface.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Ash.Domain.Verifiers.ValidateArgumentsToCodeInterface do
+  @moduledoc """
+  Validate the arguments defined in the code interface
+  and reject arguments that are not action attributes/arguments
+  """
+  use Spark.Dsl.Verifier
+
+  alias Ash.Resource.Verifiers.ValidateArgumentsToCodeInterface
+
+  @impl true
+  def verify(dsl) do
+    for reference <- Ash.Domain.Info.resource_references(dsl) do
+      for interface <- reference.definitions, match?(%Ash.Resource.Interface{}, interface) do
```

Comment:
This check might not be necessary, but I'm not sure if definitions only ever contain `Resource.Interface`s

Reply:
It can contain `Ash.Resource.CalculationInterface` as well, so your intuition here was correct.